hackathon_site/event/models.py

```diff
@@ -27,7 +27,7 @@ def __str__(self):
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     team = models.ForeignKey(
-        Team, related_name="profiles", on_delete=models.CASCADE, null=False
+        Team, related_name="profile", on_delete=models.CASCADE, null=False
```

Comment:
Two things:
- This should stay as profiles. Since this is a foreign key, we know that the relationship between Profiles and Teams is many to one, therefore team to profile is one to many. The `related_name` sets up the reverse relationship property on teams - which we want to be `some_team.profiles`
- In the future, any time you modify a model file, you have to generate a migration with `python manage.py makemigrations` (not in this case, since this change will be reverted). That will provide the database with instructions to upgrade the schemas, and in this case things like `related_name` are also stored in migrations to build a history of the model.

Reply:
Should be updated now